server/zanata-frontend/src/app/editor/components/MTSuggestionsButton/index.js

```diff
@@ -0,0 +1,24 @@
+import Button from 'antd/lib/button'
+import 'antd/lib/button/style/css'
+import Tooltip from 'antd/lib/tooltip'
+import 'antd/lib/tooltip/style/css'
+import React from 'react'
+
+class MTSuggestionsButton extends React.Component {
```

Comment:
The component should take a `backendId` prop. If backendId = 'GOOGLE' we show the Google markup below. Otherwise, we use plain text `Translated by ${backendId}`